src/styles/components/SearchAutocomplete.scss

```diff
@@ -0,0 +1,93 @@
+.search-autocomplete {
```

Comment:
Id recommend using the css-modules way of css. See: https://github.com/quran/quran.com-frontend/blob/master/src/containers/Home/index.js#L11

and file: https://github.com/quran/quran.com-frontend/blob/master/src/containers/Home/style.scss